card/db/migrate_core_cards/20141119001955_make_symlinks_relative.rb

```diff
@@ -2,7 +2,7 @@
 
 class MakeSymlinksRelative < Card::Migration
   def up
-    Cardio.paths['files'].each do |file_path| 
+    Card.paths['files'].each do |file_path| 
```

Comment:
Most of these worked to convert, but there were a couple that had to stay with Cardio.

You'll also see that some of the paths for migrations moved to Cardio too so they wouldn't reference Card before loading the class.